spring-cloud-open-service-broker-contract-tests/build.gradle

```diff
@@ -18,12 +18,14 @@ import org.springframework.cloud.contract.verifier.config.TestFramework
 
 buildscript {
 	dependencies {
+		classpath("io.spring.gradle:dependency-management-plugin:1.0.9.RELEASE")
```

Comment:
Just curious, why use the dm plugin here?

Reply:
Spring Cloud Contract seems to have some dependency on the DM plugin. When I tried to use the platform dependency, the submodule would not build. Considering this is only for tests and we aren't publishing this module, I decided to leave it alone.